dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/MetadataDumper.java

```diff
@@ -265,8 +270,34 @@ private boolean checkRequiredTaskSuccess(
     return true;
   }
 
+  private void outputCorrectLogStartAndEndDates(
+      SummaryLinePrinter linePrinter, ConnectorArguments connectorArguments) {
+
+    ZonedDateTime queryLogStartDate = connectorArguments.getQueryLogStart();
+    ZonedDateTime queryLogEndDate = connectorArguments.getQueryLogEnd();
+    ZonedDateTime actualQueryLogStartDate = QueryLogDates.getQueryLogStartDate();
+    ZonedDateTime actualQueryLogEndDate = QueryLogDates.getQueryLogStartDate();
+
+    DateTimeFormatter DATE_FORMAT =
```

Comment:
It looks like this object should be defined as `static` and reused across multiple method calls.
Otherwise, if you prefer to keep it local and lazy, I'd suggest moving it under the `if` statement below.